src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -146,11 +146,6 @@ void OnSetImageSource(UIImage? image)
 
 		public static Task MapImageSourceAsync(IButtonHandler handler, IImage image)
 		{
-			if (image.Source == null)
```

Comment:
This is the main change. This prevents that once the image is set it can be updated with something null.